app/controllers/declaration/TransportContainerController.scala

```diff
@@ -165,13 +184,15 @@ class TransportContainerController @Inject()(
   private def containerId(values: Seq[String]): String = values.headOption.getOrElse("")
 
   private def updateCache(updatedContainers: Seq[Container])(implicit r: JourneyRequest[AnyContent]): Future[Option[ExportsDeclaration]] =
-    updateExportsDeclarationSyncDirect(model => model.copy(containerData = Some(TransportInformationContainerData(updatedContainers))))
+    updateExportsDeclarationSyncDirect(
+      model => model.copy(transportData = Some(model.transportData.getOrElse(new TransportData).copy(containers = updatedContainers)))
```

Comment:
can we move this `copy` to model ?

Reply:
Moved.  Also created similar method for updating TransportPayment